en/docs/learn/extensions/managing-workflow-extension/configuring-workflows-for-tenants.md

```diff
@@ -163,9 +163,8 @@ To deploy BPELs in an API Manager tenant space, the tenant space should be acces
 
 #### Creating a BPEL
 
-In this section, you create a BPEL that has service endpoints pointing to services hosted in the tenant's space. This example uses the [Application Creation](https://docs.wso2.com/api-manager/Workflow%3A+Application+Creation) workflow.
+In this section, you create a BPEL that has service endpoints pointing to services hosted in the tenant's space. This example uses the [Application Creation](https://docs.wso2.com/display/AM260/Adding+an+Application+Creation+Workflow) workflow.
```

Comment:
@shaniR , we shouldn't be pointing to AM260

Reply:
oh yes you're right. I'll fix it.